lite/examples/image_classification/ios/ImageClassification/Camera Feed/PreviewView.swift

```diff
@@ -20,6 +20,51 @@ import AVFoundation
  */
 class PreviewView: UIView {
 
+  #if targetEnvironment(simulator)
```

Comment:
Instead of using `#if` on the same file, could you create a separate file for the simulator version?
Maybe `ImagePreviewView`?

Reply:
I kept both in the same file because it requires less storyboard modification and allows the app to fall back on static images if camera configuration fails or permission is denied, let me know if this is ok.